README.md

```diff
@@ -152,6 +152,38 @@ where date_day::date >= '2019-01-01'
 group by 1
 ```
 
+### Running locally
+
+A `docker-compose` environment starts a Thrift server and a Postgres database (Hive metastore)
+
+```
+docker-compose up
+```
+
+Your profile should look like this:
+
+```
+local-spark:
+  type: spark
+  method: thrift
+  schema: dbt
+  host: 127.0.0.1
+  port: 10000
+  user: hadoop
+  connect_retries: 5
+  connect_timeout: 60
+```
+
+The Spark UI should be available at [http://localhost:4040/sqlserver/](http://localhost:4040/sqlserver/)
```

Comment:
It may be obvious to some users, but can we expand this to also reference the SQL/JDBC endpoint? Something like:

```suggestion
**Connecting to the local spark instance:**

* The Spark UI should be available at [http://localhost:4040/sqlserver/](http://localhost:4040/sqlserver/)
* The endpoint for SQL-based testing is at `http://localhost:10000` and can be referenced with the Hive or Spark JDBC drivers using connection string `jdbc:hive2://localhost:10000`
```